goodtablesio/settings.py

```diff
@@ -65,3 +65,7 @@
 
 GITHUB_CLIENT_ID = os.environ['GITHUB_CLIENT_ID']
 GITHUB_CLIENT_SECRET = os.environ['GITHUB_CLIENT_SECRET']
+
+# Sentry
+
+SENTRY_DSN = os.environ['SENTRY_DSN']
```

Comment:
`os.environ.get('SENTRY_DSN')` (see comment about making it optional)

Also please add it to `.env.example`